erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -63,6 +63,10 @@ def validate_filters(filters, account_details):
 			if not account_details.get(account):
 				frappe.throw(_("Account {0} does not exists").format(account))
 
+	if not filters.get("categorize_by") and filters.get("group_by"):
```

Comment:
There were other changed reports in your PR too, see if same logic applies there. Idk much about those.